Doppler.BigQueryMicroservice/appsettings.json

```
@@ -28,7 +28,8 @@
     "AccountName": "fromdoppler",
     "Username": "easla@fromdoppler.com",
     "FromName": "Doppler",
-    "FromAddress": "info@fromdoppler.com"
+    "FromAddress": "info@fromdoppler.com",
+    "ReplyToEmail": "support@fromdoppler.com"
```

Very minor, but...

What do you think about using the word _address_ in place of _Email_ to continue with the previous convention?

```suggestion
    "ReplyToAddress": "support@fromdoppler.com"
```

---

suggets accepted and applied